layouts/partials/announcement.html

```diff
@@ -15,7 +15,7 @@
     {{- if or (eq .endTime nil ) (gt ( time .endTime ) now ) -}}
       {{- if not $announcementShown -}}
       {{- $announcementShown = true -}}
-  <div lang="en" id="announcement" style="background-color: #3371e3; color: #fff; {{ .style | safeCSS }}">
+  <div lang="en" id="announcement" announcement-name={{ .name }} style="background-color: #3371e3; color: #fff; {{ .style | safeCSS }}">
```

Comment:
What if the `.name` contains a space?

Reply:
The token conversion replaces whitespaces in name with '_' underscores, I've tested this behaviour and it works well